fab/environments.yml

```diff
@@ -165,40 +165,45 @@ icds:
         num_processes: 1
       CaseSearchToElasticsearchPillow:
         num_processes: 1
-      CaseToElasticsearchPillow:
-        num_processes: 1
       DefaultChangeFeedPillow:
         num_processes: 1
       DomainDbKafkaPillow:
         num_processes: 1
-      FormSubmissionMetadataTrackerPillow:
-        num_processes: 1
       GroupPillow:
         num_processes: 1
       GroupToUserPillow:
         num_processes: 1
-      kafka-ucr-main:
-        num_processes: 1
       kafka-ucr-static-awc-location:
-        num_processes: 1
-      kafka-ucr-static-cases:
         num_processes: 4
```

Comment:
Awc-location only needs one process. it only listens to locations and it's totally caught up. having 4 is overkill